From 95e816a3a2cf6cca963668d31268723ce84cb04e Mon Sep 17 00:00:00 2001
From: Qiang Yu <yuq825@gmail.com>
Date: Thu, 24 May 2018 14:22:03 +0800
Subject: [PATCH 27/44] drm/lima: Kconfig only depend on ARM or ARM64

The depend list gets longer and longer. When adding
ARCH_ZYNQMP I decide to just depend on ARM or ARM64
as this GPU can be seen on many ARM SoCs.

Signed-off-by: Qiang Yu <yuq825@gmail.com>
---
 drivers/gpu/drm/lima/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/lima/Kconfig b/drivers/gpu/drm/lima/Kconfig
index 7256625..870f66f 100644
--- a/drivers/gpu/drm/lima/Kconfig
+++ b/drivers/gpu/drm/lima/Kconfig
@@ -4,7 +4,7 @@
 config DRM_LIMA
        tristate "LIMA (DRM support for ARM Mali 400/450 GPU)"
        depends on DRM
-       depends on ARCH_SUNXI || ARCH_ROCKCHIP || ARCH_EXYNOS || ARCH_MESON
+       depends on ARM || ARM64 || COMPILE_TEST
        select DRM_SCHED
        select DRM_TTM
        help
-- 
2.7.4

